code-review-tester-service/Program.cs

```diff
@@ -3,4 +3,10 @@
 
 app.MapGet("/", () => "I'm ALIVE!");
 
+app.MapGet("/tests/", (string? level) => 
+{
```

Comment:
You're not validating anything here. I think easy, medium, hard, etc should be an enum that can be validated. If the user requests something else ("blue") it should fail.